x-pack/plugins/cases/public/components/case_view/index.tsx

```diff
@@ -463,34 +443,11 @@ export const CaseComponent = React.memo<CaseComponentProps>(
                       renderInvestigateInTimelineActionComponent={
                         timelineUi?.renderInvestigateInTimelineActionComponent
                       }
+                      statusActionButton={statusActionButton}
```

Comment:
I think there is no need to pass the button as a prop here. You can easily construct the button inside `AddComment` or `UserActionTree`.

Reply:
Let's leave it for another PR.